lot/views.py

```diff
@@ -67,6 +72,9 @@ def buy_it_now(self, request, pk=None):
             auction.current_price = buy_it_now_price
             auction.auction_status = Status.CLOSED
             auction.save(update_fields=['current_price', 'auction_status'])
+            if user_email != '':
+                send_lot_sold_email_task(lot, user_email)
```

Comment:
плюс в этой функции это можно поставить перед 
app.control.revoke(task_id=f'{TASK_NAME_CLOSE_AUCTION}_{auction.id}')
```suggestion
            if user_email:
                send_lot_sold_email_task(lot, user_email)
```